README.md

```diff
@@ -1,35 +1,32 @@
-# Teamscale JaCoCo Agent
+# Teamscale JaCoCo Agent [![Build Status](https://travis-ci.com/cqse/teamscale-jacoco-agent.svg?branch=master)](https://travis-ci.com/cqse/teamscale-jacoco-agent)
 
-## Contributing
+## Download
 
-- Please create a GitLab issue for all changes
-- Use merge requests. Use the "Definition of Done" description template for every merge request.
-- There's a Teamscale project, please fix all findings before submitting your
-  merge request for review. The Teamscale coding guidelines and Definition of Done
-  apply as far as possible with the available tooling.
-- After merging, please tag the merge commit with the version number, e.g. `v8.1.0`
+* [Binary Distribution](https://github.com/cqse/teamscale-jacoco-agent/releases)
+* [Docker Container](https://hub.docker.com/r/cqse/teamscale-jacoco-agent/tags/)
 
-## Downloading
+## Documentation
 
-We use Gitlab CI to build the distribution zip. Any successful build will have it in its
-build artifacts under `build/distributions`. Download it from there.
+* [Teamscale JaCoCo Agent](agent/USAGE.md)
+* [Java Web Start Wrapper](javaws-wrapper/README.md)
 
-In case Gitlab has pruned the artifacts (happens after 1 week) you can simply re-run the
-build.
+## Development
 
-__Use only zips produced by builds for a version tag (Tags tab in Gitlab). This ensures
-that we always know which exact version a customer has deployed.__
+### Contributing
 
-## Publishing
+* Create a GitHub issue for changes
+* Use pull requests. Complete the "Definition of Done" for every pull request.
+* There's a Teamscale project, please fix all findings before submitting your pull request for review. The Teamscale coding guidelines and Definition of Done apply as far as possible with the available tooling.
 
-All tags are built automatically. Only use builds from tagged commits. This ensures that
-we always know which code a customer is using.
+### Publishing
 
-The build server will build the project and you can download a build artifact with the dist afterwards.
-If the artifcats have been removed already, trigger the build again in GitLab.
+After merging, please create a GitHub Release tag with the version number, e.g. `v8.1.0`
```

Comment:
Please mention that we use semantic versioning and link to the semver website and our changelog